src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE = 5;
+    private static final int MAX_VALUE = 10;
+    private final Random random = new Random();
+    private final ColorSupplier supplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int index = random.nextInt(FIGURE);
+
+        switch (index) {
+            case 0:
+                Figure circle = new Circle(supplier.getRandomColor(), random.nextInt(MAX_VALUE));
+                System.out.println(circle.draw());
+                break;
```

Comment:
It is better to move to private method,
do the same for others
```suggestion
                return prepareCircle();
```